README.rst

```diff
@@ -49,12 +49,40 @@ Add the Glitter News URLs to your project’s URLconf as follows:
 Configuration
 =============
 
-The django-glitter-news provides just one setting that you can enable in your
-project.
+The following options can be set in your project's `settings.py`:
 
 GLITTER_NEWS_TAGS
 -----------------
 
-Default: ``False``
-
 This setting enables tags for the model ``Post`` in your project.
+
+Default is ``False``.
+
+Example:
+
+.. code-block:: python
+    GLITTER_NEWS_TAGS = True
+
+NEWS_PER_PAGE
+-------------
+
+Set the number of news articles which will be displayed on a page.
+
+Default is ``10``.
+
+Example:
+
+.. code-block:: python
+    NEWS_PER_PAGE = 10
+
+NEWS_NO_STICKY_ON_ALL
```

Comment:
Would prefer to avoid a negative setting being set to true if possible.

Maybe `NEWS_STICKY_ON_ALL`? Default to `True`, then we change it to `False` on projects which don't want it.